SurfaceDevCenterManager/Program.cs

```diff
@@ -172,12 +173,15 @@ private static async Task<ErrorCodes> MainAsync(string[] args)
             }
             else
             {
-                string loopServersString = ConfigurationManager.AppSettings["loopservers"];
-                if (loopServersString != null)
+                if (!OverrideServerPresent)
                 {
-                    string[] serversList = loopServersString.Split(',');
-                    int x = (new Random()).Next(0, serversList.Length);
-                    OverrideServer = int.Parse(serversList[x]);
+                    string loopServersString = ConfigurationManager.AppSettings["loopservers"];
+                    if (loopServersString != null)
+                    {
+                        string[] serversList = loopServersString.Split(',');
+                        int x = (new Random()).Next(0, serversList.Length);
```

Comment:
The range checked for above is: OverrideServer >= myCreds.Count 
I'm curious why that is different that range used here: serversList.Length? Could they be make the consistent?

Reply:
myCreds.Count is the number of available servers so is the right bounds check for OverrideServer.

serversList.Length is against the config parameter for the number of servers you want to pick from randomly inside a smaller specified list, not the full server list.